contracts/LiquidityBridgeContract.sol

```diff
@@ -433,7 +433,7 @@ contract LiquidityBridgeContract {
         require(block.number <= quote.expireBlocks, "LBC: Quote expired by blocks");
         require(msg.sender == quote.lpAddress, "LBC: Wrong sender");
         require(bridge.getBtcTransactionConfirmations(btcTxHash, btcBlockHeaderHash, partialMerkleTree, merkleBranchHashes) >= quote.transferConfirmations, "LBC: Don't have required confirmations");
-        payable(address(uint160(uint256(quote.derivationAddress)))).transfer(quote.valueToTransfer);
+        payable(quote.lpAddress).transfer(quote.valueToTransfer + quote.fee);
```

Comment:
minor comment: this property will be removed on PR #50 , but I can adjust it on that PR since this is not the purpose of this PR